docs/extras/beta/index.md

```diff
@@ -0,0 +1,79 @@
+
+### API
+
+The API is currently at **v1**. The base endpoint for the API is [here](http://api.opentrials.net/v1/), and interactive documentation for the API can be found [here](http://api.opentrials.net/v1/docs/).
+
+The API exposes several RESTful endpoints that allow querying the primary entities of the OpenTrials data model. This is particularly useful for creating applications that paginate through related data, or where the application has identifiers for particular objects and wants to query all information for those objects.
+
+The API also exposes a search endpoint, which is backed by Elasticsearch. This allows for deep queries into the entire database, and does not necessarily require any knowledge of the data model, or the relations between entities, to yield useful results. We've found that, given the nature of the data itself, the search endpoint is the most useful endpoint for regular use.
+
+Our API implements the [Open APIs specification](https://openapis.org). One of the many great benefits of implementing against Open APIs is that client libraries can be auto-generated from the spec itself. So, fire up a REPL in [Python](https://github.com/Yelp/bravado), [JavaScript](https://github.com/swagger-api/swagger-js) or any other language that has a Swagger/Open APIs implementation, and start playing with the data.
+
+### Explorer UI
+
+The [Explorer UI](https://explorer.opentrials.net) is the main portal into the OpenTrials data. It is based on a search-driven user experience, and once a user reaches a trial page, opens up further navigation into the related entities for that trial.
+
+The explorer easily enables users to navigate back to the data sources used to populate the database, and notably features many prompts for user contributions to help enhance the database by either identifying errors or contributing new data.
+
+We are very happy with the usability of the Explorer UI, and we hope you will be too.
+
+### Query UI
+
+The [Query UI](https://query.opentrials.net/) is a way to perform ad hoc SQL queries over the entire OpenTrials data warehouse, and then do interesting visualisations with the results of those queries.
+
+We leverage the excellent [Re:dash](http://redash.io) for this, and we've had great success in the lead up to the beta in using this to extract interesting insights out of the data.
+
+### Crowdsourcing UI
+
+The [Crowdsourcing UI](http://crowdcrafting.org/account/opentrials/) is a way to allow users to help us improve the OpenTrials database via an interactive process, either validating data matches that have been made, or by making new manual matches across different records in the database.
+
+We leverage the excellent [Crowdcrafting](https://crowdcrafting.org) for this. At beta, we are launching with a preview of this feature by exposing a "task" to validate matches we've automatically made in processing clinical trial records.
```

Comment:
I wouldn't talk about this task here, as we don't want people to actually do it. We can point them to http://crowdcrafting.org/project/opentrials-fda-indications/ instead.